parsec/serde/packing.py

```diff
@@ -28,7 +28,7 @@ def packb(data: Mapping[str, Any], exc_cls: Any = SerdePackingError) -> bytes:
     def _default(obj: Any) -> Any:
         if isinstance(obj, DateTime):
             return ExtType(1, struct_pack("!d", obj.timestamp()))
-        elif isinstance(obj, UUID):
+        elif isinstance(obj, (UUID, EnrollmentID)):
```

Comment:
`EnrollmentID` really needed ?

Reply:
Yes, thats because `UUID` and `EnrollmentID` are two different kind of instance (even if they're in fact the same). Otherwise this function will raise a `TypeError` exception.

Reply:
Why others IDs are not present there ?

Reply:
Check this PR please: https://github.com/Scille/parsec-cloud/pull/3598

Reply:
You can rebase and remove changes about `EnrollmentID`

Reply:
Done.